mod-pubsub-server/src/test/java/org/folio/rest/impl/PublishersApiTest.java

```diff
@@ -284,13 +244,7 @@ public void shouldDeletePublisherOnDelete() {
       .withEventDescriptors(Collections.singletonList(createdEventDescriptor))
       .withModuleId("test-module-1.0.0");
 
-    RestAssured.given()
-      .spec(spec)
-      .body(JsonObject.mapFrom(publisherDescriptor).encode())
-      .when()
-      .post(EVENT_TYPES_PATH + DECLARE_PUBLISHER_PATH)
-      .then().log().all()
```

Comment:
Did we loose `.log().all()` when extracting method?

Reply:
should I revert back to the previous state?

Reply:
fixed